slave/process-voms.sh

```diff
@@ -42,6 +42,22 @@ function process {
 
 	# Iterate through every VO and check which user will be added or removed
 	for VO in `echo -e $VOS`; do
+		# First, check if some part should be skipped
+		if echo ${NO_CREATE_MEMBERS} | grep -wq ${VO}; then
```

Comment:
grep's -w switch uses also '.' as a word delimiter. Therefore when I want to skip VO "test.myvo.cz" then "myvo.cz" will be also skipped.